demo-java/app/src/gms/java/com/example/placesdemo/AutocompleteAddressActivity.java

```diff
@@ -105,14 +125,14 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
         if (requestCode == AUTOCOMPLETE_REQUEST_CODE) {
             if (resultCode == RESULT_OK) {
                 Place place = Autocomplete.getPlaceFromIntent(data);
-                Log.i(TAG, "Place: " + place.getAddressComponents());
+                i(TAG, "Place: " + place.getAddressComponents());
```

Comment:
I think these are more readable as non-static imports. If this was automatically done by the IDE, there should be a way to disable that.

Reply:
Thanks for catching. Yeah this is a new behavior I started seeing in Android Studio! I thought I reverted all their "fixes" but missed this one.